client/src/game/Game.js

```diff
@@ -26,6 +32,7 @@ const player = {
   offsetX: 0,
   offsetY: 0,
   jumping: true,
+  direction: "FRONT",
```

Comment:
The idea with the enum (we talked about last PR) is to write the string like `FRONT` only once and access variable instead.

Example:

```suggestion
  direction: DIRECTIONS.FRONT
```

In this case, you would import the `DIRECTIONS` and use it everywhere in your app.

Reply:
ah okay. Then I understood it wrong, sorry 😇 . I will definitely implement it this way. Way easier 😉